public/locales/sp/translation.json

```diff
@@ -159,7 +159,7 @@
     "dependentLocality": "Localidad Dependiente",
     "sortingCode": "Código de Ordenamiento",
     "state": "Estado / Provincia",
-    "userRegistrationRequired": "Registro de usuario requerido",
+    "Is Public": "Es público",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.